[core, okx, binance, bybit] Get fee rates

rizer1980 opened the pull request:
[core]
AccountService getDynamicTradingFeesByInstrument - add Optional category param

[okx, binance, bybit]
add/fix getDynamicTradingFeesByInstrument

[binance]
add separate resilience to Future account
add/fix stream futures api

[okx stream]
merge public and private channels in one connection

rizer1980 committed:
Merge remote-tracking branch 'github.com/knowm/XChange/develop' into develop

rizer1980 committed:
Merge remote-tracking branch 'XChange-rizer-public-fork/develop' into develop

rizer1980 committed:
[core]
AccountService add optional param to getDynamicTradingFeesByInstrument
[binance, okx, bybit]
implement getDynamicTradingFeesByInstrument
[binance]
fix resilience for futures

rizer1980 committed:
[okx stream]
merge authed and public stream services in one

rizer1980 committed:
[binance stream]
fix testnet api url

rizer1980 committed:
Merge remote-tracking branch 'github.com/knowm/XChange/develop' into PR/get-fee-rates

rizer1980 committed:
[coinsph]
fix getDynamicTradingFeesByInstrument

rizer1980 commented:
@timmolter 
somethink wrong?
Or merge after Xchange release?

timmolter commented on xchange-core/src/main/java/org/knowm/xchange/service/account/AccountService.java:
```diff
@@ -221,7 +223,8 @@ default List<FundingRecord> getFundingHistory(TradeHistoryParams params) throws
    */
-  default Map<Instrument, Fee> getDynamicTradingFeesByInstrument() throws IOException {
+  default Map<Instrument, Fee> getDynamicTradingFeesByInstrument(Object... category)
+      throws IOException {
```
is there a more specific object such as String... that could be used instead of Object...

rizer1980 replied:
Yeah, I didn't think about that.

rizer1980 committed:
Merge remote-tracking branch 'github.com/knowm/XChange/develop' into PR/get-fee-rates

rizer1980 committed:
[core]
AccountService - change getDynamicTradingFeesByInstrument method optional param from Object to String

rizer1980 committed:
[core]
AccountService - change getDynamicTradingFeesByInstrument method optional param from Object to String

rizer1980 committed:
Merge remote-tracking branch 'XChange-rizer-public-fork/PR/get-fee-rates' into PR/get-fee-rates

rizer1980 committed:
Merge remote-tracking branch 'github.com/knowm/XChange/develop' into PR/get-fee-rates

# Conflicts:
#	xchange-okex/src/main/java/org/knowm/xchange/okex/OkexAdapters.java
#	xchange-stream-okex/src/test/java/info/bitrich/xchangestream/okex/OkexStreamingPublicDataIntegration.java

rizer1980 commented:
bump
